src/styles/index.scss

```diff
@@ -1,3 +1,19 @@
-body {
+@use './header.scss';
+@use './card-container';
```

Comment:
According to the code style guide, you should use double quotes in SCSS. So it should be `@use "./card-container";`